specs/paymentrequest.html

```diff
@@ -425,6 +425,13 @@
             length of 1, then set <a><code>shippingOption</code></a> to the <code>id</code> of
             the only <a><code>ShippingOption</code></a> in the sequence.
           </li>
+          <li>
+            If <code>details</code> contains a <code>shippingOptions</code> sequence with a
+            length greater than 1, and if any <a><code>ShippingOption</code></a> in the sequence
+            has the <code>selected</code> field set to <code>true</code>, then set
+            <a><code>shippingOption</code></a> to the <code>id</code> of the last <a><code>ShippingOption</code></a>
+            in the sequence with <code>selected</code> set to <code>true</code>.
+          </li>
```

Comment:
I'd be hesitant to make this a conformance issue, though. It may be perfectly reasonable for the user agent to use locally-configured preferences to override the "selected" flag; e.g., "remember my choice from last time" or "always default to the cheapest shipping option."  So I would far prefer to see this specified as a hint rather than a normative behavior.